### PR TITLE
`General`: Enhance exercise buttons with exercise type icons

### PR DESCRIPTION
**Pull request by milljoniaer**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://ls1intum.github.io/Artemis/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://ls1intum.github.io/Artemis/dev/development-process/#naming-conventions-for-github-pull-requests).

#### Client
- [x] **Important**: I implemented the changes with a very good performance, prevented too many (unnecessary) REST calls and made sure the UI is responsive, even with large data.
- [x] I followed the [coding and design guidelines](https://ls1intum.github.io/Artemis/dev/guidelines/client/).
- [x] Following the [theming guidelines](https://ls1intum.github.io/Artemis/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [x] I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://ls1intum.github.io/Artemis/dev/guidelines/client-tests/).
- [x] I added `authorities` to all new routes and checked the course groups for displaying navigation elements (links, buttons).
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context
Stephan suggested adding exercise-type icons for the Create/Import Exercise buttons in the exercise group management view to improve UX. 

### Description
These icons are already displayed on mobile, when the button label is not displayed anymore. This PR displays them also on desktop after the label.

Edit: Same goes for the course exercise create/import buttons and the order of create and import buttons is unified to first the "Create" Button and then the "Import" Button.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Exam with 1 Exercise Group
- View the control section there on mobile and desktop
- Verify that the exercise-type icons are displayed on both

#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Screenshots

Desktop:
image

Mobile:
image

course exercises:
image

### CODE REVIEW

**Review by lennart-keller — APPROVED** [review]
Code changes lgtm. Have you considered putting the icons in the front instead? Personally, I am a bit unsure what looks better:
icons

**Review by DominikRemo — APPROVED** [review]
Code LGTM
Tested on ts2 and works as expected

**Review by JohannesStoehr — CHANGES_REQUESTED** [review]
Looks good over all and I personally like the icons behind the text better as you already did.
Can you also add these icons to the course exercise page? Additionally I noticed that the import and create option in the exam page is the other way round compared to the course page. You could unify that order as well (I'd suggest creation above/before import)

**Comment by milljoniaer** [comment]
Thank you for your feedback, I sticked to the exercise icon at the end, added the icon also in the course exercise views and unified the order of create/import buttons. Please have another look at the PR.

**Review by JohannesStoehr — DISMISSED** [review]
Thanks for unifying the two occurrences.
Code looks good to me

**Review by DominikRemo — APPROVED** [review]

**Review by terlan98 — APPROVED** [review]
Tested on TS5. The icons are visible for exam exercise groups and course exercises 👍

File: src/main/webapp/app/exercises/shared/manage/exercise-create-buttons.component.ts [hunk 1]
```diff
@@ -36,4 +41,19 @@ export class ExerciseCreateButtonsComponent implements OnInit {
             this.router.navigate(['course-management', this.course.id, this.exerciseType + '-exercises', result.id, 'import']);
         });
     }
+
+    getExerciseTypeIcon(exerciseType: ExerciseType) {
```
Comment (krusche) [inline src/main/webapp/app/exercises/shared/manage/exercise-create-buttons.component.ts:45]: This code already exists in `ExamExerciseImportComponent`. Please unify it in a common util or service class

**Review by JohannesStoehr — DISMISSED** [review]
Reapprove after unification

**Review by DominikRemo — APPROVED** [review]
Tested on ts1. Works as expected

**Review by terlan98 — APPROVED** [review]
Re-tested on TS5 and reapproved

**Review by lennart-keller — APPROVED** [review]
Recent code changes lgtm

**Pull request merged**